tests/ttnn/multichip_unit_tests/test_multidevice_TG.py

```diff
@@ -196,6 +196,7 @@ def test_galaxy_matmul_2d_fracture(M, K, N, weights_dtype, cluster_shape, device
     assert out_pass
 
 
+@pytest.mark.skip("DRAM-SHARDED Matmuls gives ND PCC")
```

Comment:
Is there a TT issue made to track this issue?
Is this something that was pre-existing already on main already??

Reply:
It's only an issue on TG and just discover recently, will attach the issue number with the skip statement.